assets/Js/mobile-menu.js

```diff
@@ -0,0 +1,91 @@
+  },
+  {
+    id: 2,
+    name: 'Baldewin Luan',
+    title: 'Iusto, voluptatem, vero commodi quae omnis temporibus asperiores possimus exercitationem repellendus',
+    image: 'assets/images/featured-speakers/speaker_avatar-2.png',
+    description: 'Lorem ipsum dolor sit amet consectetur, adipisicing elit. Ex exercitationem vel vitae nostrum!',
+  },
+  {
+    id: 3,
+    name: 'Jerry S. Levan',
+    title: "daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+    image: 'assets/images/featured-speakers/speaker_avatar-3.png',
+    description: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Animi praesentium officiis voluptatem obcaecati. Iusto, voluptatem, vero commodi quae omnis temporibus asperiores possimus exercitationem repellendus optio voluptate praesentium! Voluptatum, dignissimos assumenda!',
+  },
+  {
+    id: 4,
+    name: 'Lidya Nasih',
+    title: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+    image: 'assets/images/featured-speakers/speaker_avatar-4.png',
+    description: 'Lorem ipsum dolor sit amet consectetur, adipisicing elit. Ex exercitationem vel vitae nostrum!',
+  },
+  {
+    id: 5,
+    name: 'Oliver Horvatinčić',
+    title: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+    image: 'assets/images/featured-speakers/speaker_avatar-5.png',
+    description: 'Lorem ipsum dolor sit amet consectetur, adipisicing elit. Ex exercitationem vel vitae nostrum!',
+  },
+  {
+    id: 6,
+    name: 'Jian Jen',
+    title: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+    image: 'assets/images/featured-speakers/speaker_avatar-6.png',
+    description: 'Lorem ipsum dolor sit amet consectetur, adipisicing elit. Ex exercitationem vel vitae nostrum!',
+  },
+];
+function addFeaturedSpeaker() {
+  const speakers = Object.keys(featuredSpeaker);
+  speakers.forEach((speaker) => {
+    const card = document.createElement('div');
+    card.classList.add('speaker-card');
+    featuredSpeakerContainer.append(card);
+    card.innerHTML = `<div class="speaker-img">
+        <img src="${featuredSpeaker[speaker].image}" alt="Speakers photo" class="speaker-avatar">
+      </div>
+      <div class="speaker-description-container">
+        <h4 class="speaker-name" id="speaker-name">${featuredSpeaker[speaker].name}</h4>
+          <p class="speaker-description primary"  id="speaker-qualification">${featuredSpeaker[speaker].title}</p>
+            <div class="row"><hl class="divider-dark"></hl></div>
+              <p class="speaker-description dark">${featuredSpeaker[speaker].description}</p>
+            </div>`;
+  });
+}
+
+window.onload = () => {
+  addFeaturedSpeaker();
+};
```

Comment:
- [Optional]  Dear @FevenSeyfu, Great job on using the window `onload` event, But for the manipulation of DOM object better use the `DOMContentLoaded` event of the document object. This will make sure your DOM manipulation starts once every DOM element is rendered. Right now line 76 code is throwing an error `featuredSpeakerContainer.append(card)` because `speaker-card` was not found.

![Screenshot 2023-01-14 140831](https://user-images.githubusercontent.com/69077061/212469018-16e4f36b-80cb-4fe1-be6b-94c6cf1b212e.png)

Reply:
Thank you for your feedback as per your suggestion I have  used DOMContentLoaded.